storage_controller/src/service.rs

```diff
@@ -3958,6 +3991,9 @@ impl Service {
         let import_failed = import.completion_error().is_some();
 
         if !import_failed {
+            self.activate_timeline_post_import(&import).await?;
+            tracing::info!("Post import shard reset complete");
+
```

Comment:
1. Log message suggestion
2. The loop below that waits for timeline shards to activate is redundant now that we explicitly call in for activation? Just make the `active_timeline_post_import` endpoint impl in Pageserver idempotent and get rid of the loop below ?

Reply:
Made the handler idempotent in https://github.com/neondatabase/neon/pull/11855/commits/8fa36e45e734c0a9c7b504451d32ea6971235e11